go/loopback/main.go

```diff
@@ -32,49 +37,72 @@ func Init(homeDir string, logFile string, runModeStr string, serverURI string, a
 			fmt.Println("Error decoding run mode", err, runModeStr)
 		}
 		config := libkb.AppConfig{HomeDir: homeDir, LogFile: logFile, RunMode: runMode, Debug: true, LocalRPCDebug: "Acsvip", ServerURI: serverURI, SecurityAccessGroupOverride: accessGroupOverride}
-		err = libkb.G.Configure(config, usage)
+		err = kbCtx.Configure(config, usage)
 		if err != nil {
 			panic(err)
 		}
 
-		service := (service.NewService(g, false))
-		service.StartLoopbackServer()
-		service.G().SetService()
+		svc := service.NewService(kbCtx, false)
+		svc.StartLoopbackServer()
+		kbCtx.SetService()
+		kbCtx.SetUIRouter(service.NewUIRouter(kbCtx))
 
 		serviceLog := config.GetLogFile()
 		logs := libkb.Logs{
 			Service: serviceLog,
 		}
 
 		logSendContext = libkb.LogSendContext{
-			Contextified: libkb.NewContextified(service.G()),
+			Contextified: libkb.NewContextified(kbCtx),
 			Logs:         logs,
 		}
 
+		// Uncomment after we KBFS is ready
+		// kbfsParams := libkbfs.DefaultInitParams(kbCtx)
```

Comment:
Okay, I see. Well, I'm okay with it in the interest of getting stuff working; I guess we'll have to figure out a solution to the general vendoring problem long-term...